src/main/java/org/jabref/logic/layout/format/MarkdownFormatter.java

```diff
@@ -0,0 +1,41 @@
+package org.jabref.logic.layout.format;
+
+import java.util.List;
+import java.util.Objects;
+
+import org.jabref.logic.layout.LayoutFormatter;
+
+import com.vladsch.flexmark.ext.gfm.strikethrough.StrikethroughExtension;
+import com.vladsch.flexmark.ext.gfm.tasklist.TaskListExtension;
+import com.vladsch.flexmark.html.HtmlRenderer;
+import com.vladsch.flexmark.parser.Parser;
+import com.vladsch.flexmark.util.ast.Node;
+import com.vladsch.flexmark.util.data.MutableDataSet;
+
+public class MarkdownFormatter implements LayoutFormatter {
+
+    private final Parser parser;
+    private final HtmlRenderer renderer;
+
+    public MarkdownFormatter() {
+        MutableDataSet options = new MutableDataSet();
+        options.set(Parser.EXTENSIONS, List.of(
+                StrikethroughExtension.create(),
+                TaskListExtension.create()
+        ));
+
+        parser = Parser.builder(options).build();
+        renderer = HtmlRenderer.builder(options).build();
+    }
+
+    @Override
+    public String format(final String fieldText) {
+        Objects.requireNonNull(fieldText, "Field Text should not be null, when handed to formatter");
+
+        Node document = parser.parse(fieldText);
+        String html = renderer.render(document);
+
+        // workaround HTMLChars transforming "\n" into <br> by returning a one liner
```

Comment:
is the `HTMLChars` formatter still necessary or is the HtmlRenderer already taking care of it?

Reply:
It's needed to be checked whether the HtmlRenderer does some "Magic". Without guessing, I would gess not...

The HTMLChars formatter does:

![grafik](https://user-images.githubusercontent.com/1366654/78255854-23d38e80-74f8-11ea-97f5-d3cec674745c.png)